backend/api/Models/Concepts/AcquisitionFile/AcquisitionFileMap.cs

```diff
@@ -16,7 +16,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.MinistryProjectName, src => src.MinistryProjectName)
                 .Map(dest => dest.AssignedDate, src => src.AssignedDate)
                 .Map(dest => dest.DeliveryDate, src => src.DeliveryDate)
-                .Map(dest => dest.AcquisitionFileStatusTypeCode, src => src.AcquisitionFileStatusTypeCodeNavigation)
+                .Map(dest => dest.FileStatusTypeCode, src => src.AcquisitionFileStatusTypeCodeNavigation)
```

Comment:
Fixing backend mappings broken with the refactoring to a common `FileModel` last sprint